src/blocks/donate/streamlined.js

```diff
@@ -60,19 +60,20 @@ const getAmount = formValues => {
 	return parseFloat( formValues.amount );
 };
 
+export const computeFeeAmount = ( amount, feeMultiplier, feeStatic ) => {
+	return parseFloat(
+		( ( ( amount + feeStatic ) / ( 100 - feeMultiplier ) ) * 100 - amount ).toFixed( 2 )
```

Comment:
I think this math is slightly off, if I'm not mistaken. Are the default values (`2.9` and `0.3`) coming from Stripe's [integrated pricing](https://stripe.com/pricing)? If so, shouldn't this calculation be:

```suggestion
		( amount * ( feeMultiplier / 100 ) + feeStatic ).toFixed( 2 )
```

Given a donation amount of $100, the fee should be:

Multiplier: 2.9% of $100 = $2.90
Static: $0.30
Total fee: $3.20

Right?

Reply:
Aha, that makes sense. Thanks for explaining.